api/resolvers/item.js

```diff
@@ -553,6 +553,17 @@ export default {
 
       return res
     },
+    fetchDocument: async (parent, { url }, { models }) => {
+      try {
+        const response = await fetch(ensureProtocol(url))
+        const html = await response.text()
+        if (url.includes('peertube.tv')) {
+          return html.match(/"([^"]*\/embed\/[^"]*)"/)[0].substring(1, html.match(/"([^"]*\/embed\/[^"]*)"/)[0].length - 1)
```

Comment:
This is the parse for the peertube html, it looks for the embed link and then trims the `" "` on either end

Reply:
From my brief investigation becauses `bitcoin.tv` is based on peertube it should have the exact same format so we should be able to use this exact code for parsing links from there as well